system/jlib/jthread.cpp

```diff
@@ -401,6 +401,12 @@ void Thread::startRelease()
                 pthread_attr_setstacksize(&attr, LINUX_STACKSIZE_CAP);
 #endif
         }
+        sched_param param;
+        pthread_attr_getschedparam(&attr, &param);
+        param.sched_priority = 0;
+        pthread_attr_setschedparam(&attr, &param);
+        pthread_attr_setschedpolicy(&attr, SCHED_OTHER);
```

Comment:
If the process has been started with a different riority, and therefore main is at that priority,
does this set new threads to the same priority as the process/main thread, or to a system standard (SCHED_NORMAL) priority?

Reply:
Testing suggests it ends up same priority as the main thread.